first-party/gradlew

```diff
@@ -138,19 +154,19 @@ if $cygwin ; then
         else
             eval `echo args$i`="\"$arg\""
         fi
-        i=$((i+1))
+        i=`expr $i + 1`
```

Comment:
do we need to propagate any of these changes to `jib/gradlew`?

Reply:
The main jib repo already upgraded the wrapper to 6.0.1 in https://github.com/GoogleContainerTools/jib/pull/2952. I expect the wrapper should be same.

Reply:
The wrapper is the same for the most part but there are just one or two changes here that are not present there?

Reply:
The different between the jib repo and this repo may be that this repo is upgrading from `gradle-5.2.1-all.zip` while the jib repo is from `gradle-5.2.1-bin.zip`. (Or maybe not.) In any case, these changes are after running `./gradlew wrapper --gradle-version=6.0.1`, so this should be right.

Reply:
Ah I see, sounds good.